### ISSUE TITLE
Loss function of WGAN

### ISSUE THREAD

**Issue by WuKongy** [wuhuikai/GP-GAN#21]
A negative sign is missing? 
updater.py line 62: Maybe the loss = (1 - self.args.l2_weight) * errG + self.args.l2_weight * l2_loss  should be changed to " **-** (1 - self.args.l2_weight) * errG + self.args.l2_weight * l2_loss"

**Comment by wuhuikai** [comment]
Since the `d_loss` in line 15 is `errD_real - errD_fake`, I think line 62 is correct.
